code/modules/antagonists/pirate/pirate_event.dm

```diff
@@ -22,9 +22,15 @@
 /datum/round_event/pirates/start()
 	send_pirate_threat(chosen_gang)
 
-/proc/send_pirate_threat(datum/pirate_gang/chosen_gang)
+/proc/send_pirate_threat(datum/pirate_gang/chosen_gang, is_heavy)
 	if(!chosen_gang)
-		chosen_gang = pick_n_take(GLOB.pirate_gangs)
+		if(!is_heavy)
+			chosen_gang = pick_n_take(GLOB.light_pirate_gangs)
+		else
+			chosen_gang = pick_n_take(GLOB.heavy_pirate_gangs)
```

Comment:
:-1: We should avoid boolean parameters when they're like this.

I think instead we should expect consumers to pass in the list of pirates to roll from. That way we also get "light or heavy" pirate rolling for free, which we might want for something like the event.

Reply:
By that I mean `send_pirate_event(GLOB.light_pirate_gangs + GLOB.heavy_pirate_gangs)` would Just Work

Reply:
big agree with mothblocks